src/options.rs

```diff
@@ -118,6 +122,26 @@ impl<'a> Options<'a> {
         }
     }
 
+    /// Change [`self.tab_width`]. The tab width is how wide
+    /// a tab character should be considered. By default, a tab
+    /// character to be consiered to have a width of 0.
```

Comment:
I'm actually thinking that we should use a more traditional width of 8. If we go through the effort of adding support for this, people should see it work out of the box — if people discover that they have unexpected TAB characters in their strings, they can either filter them out or set the width back to zero.

Reply:
I'd be fine with that; it's just a default anyways, and changing it is simple. How do you feel about this given https://github.com/mgeisler/textwrap/pull/490#discussion_r1018629643 though? If we set it to anything greater than 1, we loose the fast in `wrap_single_line` by default, at least with my current fix.